SoObjects/Mailer/SOGoMailFolder.m

```diff
@@ -1425,15 +1424,15 @@ - (NSString *) _sogoACLsToIMAPACLs: (NSArray *) sogoAcls
 
 - (NSString *) _sogoACLUIDToIMAPUID: (NSString *) uid
 {
-  SOGoGroup *group;
+  NSDictionary *dict;
   SOGoUser *user;
 
   if ([uid isEqualToString: defaultUserID])
     return uid;
 
-  group = [SOGoGroup groupWithIdentifier: uid
-                                inDomain: [[context activeUser] domain]];
-  if (group)
+  dict = [[SOGoUserManager sharedUserManager] contactInfosForUserWithUIDorEmail: uid];
```

Comment:
Respect the domain by using `[SOGoUserManager contactInfosForUserWithUIDorEmail:inDomain:]`.

Reply:
Resolved in commit 948c89f